synchronization/src/main/kotlin/de/cyface/synchronization/SyncPerformer.kt

```diff
@@ -317,6 +317,12 @@ internal class SyncPerformer(private val context: Context, private val fromBackg
                 Result.UPLOAD_SKIPPED
             }
 
+            is NoLocationData -> {
+                syncResult.stats.numSkippedEntries++
+                Log.d(TAG, e.message!!)
+                Result.UPLOAD_SKIPPED
+            }
```

Comment:
The Digural WebdavUploader now throws this exception, which needs to be handled here softly.